src/aioswitcher/device/__init__.py

```diff
@@ -35,23 +35,24 @@ class DeviceCategory(Enum):
 class DeviceType(Enum):
     """Enum for relaying the type of the switcher devices."""
 
-    MINI = "Switcher Mini", "030f", DeviceCategory.WATER_HEATER
-    POWER_PLUG = "Switcher Power Plug", "01a8", DeviceCategory.POWER_PLUG
-    TOUCH = "Switcher Touch", "030b", DeviceCategory.WATER_HEATER
-    V2_ESP = "Switcher V2 (esp)", "01a7", DeviceCategory.WATER_HEATER
-    V2_QCA = "Switcher V2 (qualcomm)", "01a1", DeviceCategory.WATER_HEATER
-    V4 = "Switcher V4", "0317", DeviceCategory.WATER_HEATER
-    BREEZE = "Switcher Breeze", "0e01", DeviceCategory.THERMOSTAT
-    RUNNER = "Switcher Runner", "0c01", DeviceCategory.SHUTTER
-    RUNNER_MINI = "Switcher Runner Mini", "0c02", DeviceCategory.SHUTTER
+    MINI = "Switcher Mini", "030f", 1, DeviceCategory.WATER_HEATER
+    POWER_PLUG = "Switcher Power Plug", "01a8", 1, DeviceCategory.POWER_PLUG
+    TOUCH = "Switcher Touch", "030b", 1, DeviceCategory.WATER_HEATER
+    V2_ESP = "Switcher V2 (esp)", "01a7", 1, DeviceCategory.WATER_HEATER
+    V2_QCA = "Switcher V2 (qualcomm)", "01a1", 1, DeviceCategory.WATER_HEATER
+    V4 = "Switcher V4", "0317", 1, DeviceCategory.WATER_HEATER
+    BREEZE = "Switcher Breeze", "0e01", 2, DeviceCategory.THERMOSTAT
+    RUNNER = "Switcher Runner", "0c01", 2, DeviceCategory.SHUTTER
+    RUNNER_MINI = "Switcher Runner Mini", "0c02", 2, DeviceCategory.SHUTTER
 
     def __new__(
-        cls, value: str, hex_rep: str, category: DeviceCategory
+        cls, value: str, hex_rep: str, protocol_type: int, category: DeviceCategory
```

Comment:
what do protocol_type stands for?
can we have an enum instead?

Reply:
Yes, can you suggest something that will reflect the two protocol types? also note that we later had classes that are named Type1 and Type2

Reply:
No, I actually can't think of anything...